module/Application/view/layout/layout.phtml

```diff
@@ -55,14 +59,14 @@
     ]); ?>
     <?php $this->headLink([
         'as' => 'font', 'rel' => 'preload', 'type' => 'font/ttf', 'crossorigin' => 'anonymous',
-        'href' => '/fonts/gewisicons/gewisicons.ttf?37l19n='
+        'href' => '/fonts/gewisicons/gewisicons.ttf?37l19n'
     ]); ?>
     <?php $this->headLink([
-        'as' => 'image', 'rel' => 'preload', 'type' => 'image/svg+xml', 'crossorigin' => 'anonymous',
```

Comment:
Why is the crossorigin removed? Shouldn't links have this attribute?

Reply:
This bug report on Chromium seems related: https://bugs.chromium.org/p/chromium/issues/detail?id=1065069

Reply:
I had that one open for a while as well as a Stackoverflow for that issue

SVG resources are documents in HTML (as opposed to images). Hence, you can interact with their contacts as with an embedded HTML. The way the SVG is included enforces it to be included later like that. I removed it entirely since 'use-credentials' did not seem appropriate and the only two allowed values are 'anonymous' and the former. My testing showed that Chrome accepts this and does not do a second request for the two SVGs.

This actually allows for interesting things such as modifying SVG colours with JavaScript so they are current origin objects. With this fix, preloaded resources are used

Reply:
Well, I suppose crossorigin doesn't matter a lot since it's hosted on the same domain.